trunner/config.py

```diff
@@ -43,6 +43,7 @@ class ConfigParser:
     class MainConfig:
         targets: Optional[Set] = None
         type: Optional[str] = None
+        should_reboot: bool = False
```

Comment:
Let's be consistent with other short names. I think thanks to specifying type the purpose is clear.

```suggestion
        reboot: bool = False
```